puppet/modules/slave/manifests/init.pp

```diff
@@ -55,6 +55,13 @@
     ensure  => absent,
   }
 
+  file { "${homedir}/.bundle/config":
```

Comment:
I think you may need to ensure the directory as well. Otherwise you get errors about that.